backend/coins/eth/rpcclient/rpcclient.go

```diff
@@ -51,6 +51,8 @@ type Interface interface {
 	// SuggestGasPrice retrieves the currently suggested gas price to allow a timely
 	// execution of a transaction.
 	SuggestGasPrice(ctx context.Context) (*big.Int, error)
+	// FeeTargets returns EIP-1559 compatible priorities (maxFeePerGas + baseFee)
+	// If not be reachable fallback to only one priority, estimated by the eth_gasPrice RPC endpoint.
```

Comment:
Remove this line - the fallback is not implemented by the rpcclient (i.e. etherscan).

Reply:
i just realized that this interface is only for RPC methods, and etherscan can provide the eth full node RPC methods, or any other full node can. 

FeeTargets() is not an RPC method but solely a etherscan feature. 

As we only use Etherscan right now for RPC, this did not become obvious :P

We can just leave it like this for now, too much hassle to restructure the code to make it "proper" when this RPC abstraction is not even used at the moment. Could consider just deleting the interface (but not in this PR).

Reply:
yeah, I know, I didn't like adding it to the interface and under `// ----- RPC node proxy methods follow` when it's not an RPC but a custom etherscan feature, I meant to leave a comment about that somewhere but I guess I didn't

I'll leave it but make a comment there for future reference